definitions/browser-application/relationships.yml

```diff
@@ -0,0 +1,10 @@
+# Select entityGuid, requestUrl from AjaxRequest where requestUrl is not null
+relationships:
+  - instrumentation.provider: newrelic
+    eventType: AjaxRequest
+    relationship: CALLS
+    target:
+      domain: EXT
+      type: SERVICE
+      domainId: requestUrl
+      processor: hostExtractor
```

Comment:
I'm wondering if the domainId should be an object. Since the processor is needed in order to obtain it.
Like:
```
domainId:
  attribute: requestUrl
  processor: hostExtractor
```

We can keep the simpler form ` domainId: requestUrl` when no processor is rquired.
Thoughts?

Reply:
ya that makes sense. it more clearly links the processor strictly to the domainID as actually intended

Reply:
couldn't quite figure out how to make it play nice with both structures, but i modified domainID to be an object that accepts processor and attribute!